### ISSUE TITLE
Design: Log Analysis Overview Enhancements & Data Latency Graph

### ISSUE THREAD

**Issue by alexmylonas** [panther-labs/panther#1257]
### Description

We want to have an additional graph to show the average data latency by log type.

More info on #1191

Provided example: 

<img width="952" alt="68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3565353030366464663035346139653930623632623831322f63666536386464622d643963312d343263662d396438372d633161393334363665656130" src="https://user-images.githubusercontent.com/14179917/88791019-baac1100-d1a1-11ea-9587-690792f7c94e.png">

### Acceptance Criteria

- Design a new bar chart or iterate on existing ones
- Incorporate chart to Log Analysis Overview mock

**Comment by dimitriszoitas** [comment]
Since the overview page is getting taller with adding so much stuff, I suggest 2 versions. The second one is using a tab for the events by log type / Data Latency by log type

[Alternative 1](https://share.goabstract.com/b6968290-2912-4a71-a564-a4bfb94078b0?mode=design&sha=0e35e08a9c4e2815e81870d1210b7ed6d8ab94e5)

[Alternative 2](https://share.goabstract.com/3731dd84-5660-45f0-b421-6f814068a5f1?mode=design&sha=0e35e08a9c4e2815e81870d1210b7ed6d8ab94e5)

**Comment by dimitriszoitas** [comment]
I've revised the Log analysis overview page, and replaced the bar chart with a line graph as @jacknagz suggested. You can find the collection [here](https://share.goabstract.com/a4c92df8-8a36-49a3-a102-01d9e8eef5f4)

**Comment by 3nvi** [comment]
Left  my thoughts as well 😄

**Comment by 3nvi** [comment]
@dimitriszoitas Still waiting for answers in this cross-post https://share.goabstract.com/0a7c4880-bf24-4fbd-bca2-c4f701ea7fad?commentId=4a53df94-3f15-413d-82b0-b98d5dd54237&sha=58f917ed6e4b286993b902400cd04d38d0fde7c6

**Comment by 3nvi** [comment]
Requirements are satisfied, this can now be closed

**Comment by jacknagz** [comment]
@3nvi @dimitriszoitas why did we switch to a line from a bar, as this originally asked? The line graph does not accurately depict what we are looking for.

**Comment by dimitriszoitas** [comment]
@jacknagz in the first mockups, I did have a bar chart. During a standup you suggested we should replace the bar chart with a line chart and that's why I changed it. The reason was mainly because there could be 15-20 log types in that bar chart and it didnt look good.  

The initial mockup can be found [here](https://share.goabstract.com/b6968290-2912-4a71-a564-a4bfb94078b0?mode=design&sha=0e35e08a9c4e2815e81870d1210b7ed6d8ab94e5)

If we do want to depict the latency with bars, we can do that but we'll have to limit the log types shown.
